### DISCUSSION TITLE
How to presert upper case in links hashes? Like path.html#L42?

### DISCUSSION THREAD

**Discussion by antonmedv** [facebook/docusaurus#5452] [Q&A] [answered]
I'd like to add links to GH file line via `#L42`, but all hashes lowercased `#l42` and don't work on github. Is there a way?

**Comment by slorber** [comment, marked as answer]
I tested using a link in both React or Markdown:

```jsx
              <Link
                className="button button--info"
                to="https://github.com/facebook/docusaurus/blob/main/website/babel.config.js#L7">
                <Translate>Playground</Translate>
              </Link>
```


```md 
[test](https://github.com/facebook/docusaurus/blob/main/website/babel.config.js#L7)
```

In both case, the links weren't lowercased and GitHub highlighted the line.

Are you sure about this behavior? Can you show how you created those links? Any open-source site online I can see (source+site)

**Reply by antonmedv** [reply]
You are right. Sorry for bothering and thanks for help.
